### PR TITLE
Next round of changes based on discussion with Justin

### PR DESCRIPTION
**Pull request by DesignByOnyx**
This includes a host of changes as discussed with Justin.

### CODE REVIEW

File: docs/index.html [hunk 1]
```diff
@@ -31,128 +26,90 @@
...
+								this.setState({
+								  todos: this.state.todos.concat({
+								    title: 'Dishes',
+								    complete: false
+								  })
+								});
+
+								// Set the first todo as complete
+								const clone = this.state.todos.slice();
+								clone[0].complete = true;
```
Comment (christopherjbaker) [inline docs/index.html]: *actually*, you're still modifying state here. to do it properly, you'd need this (also using spread cause its better =P):
```
const todos = [ ...todos ];
todos[0] = {
	...todos[0],
	complete: true,
};

this.setState({ todos });
```

File: docs/index.html [hunk 2]
```diff
@@ -31,128 +26,90 @@
...
-								}
+								<pre><code class="language-js">
+									// Add new todo
+									this.state.todos.push({
+									  title: 'Dishes',
+									  complete: false
+									});
+
+									// Set the first todo as complete
+									this.state.todos[0].complete = true;
```
Comment (christopherjbaker) [inline docs/index.html]: see above

File: docs/index.html [hunk 3]
```diff
@@ -224,16 +182,22 @@ <h3>ylem</h3>
...
 								      &lt;div&gt;
+								      	/* Pass state to children */
 								        &lt;NextPrev paginate={this.state} /&gt;
 								        &lt;Grid paginate={this.state} /&gt;
 								      &lt;/div&gt;
 								    );
 								  }
 								}
 								</code></pre>
+								<div style="padding: 195px 0; color: #ccc; text-align: center">
```
Comment (christopherjbaker) [inline docs/index.html]: for zooming and future-forward purposes, probably better to use `em` here instead of `px`

Comment (DesignByOnyx) [reply docs/index.html]: I'm an `em`-only type of guy, but if a project isn't using it then I don't either.

File: docs/index.html (hunk 1 above)
Comment (DesignByOnyx) [reply docs/index.html]: I was avoiding ...spread to make this more palatable for the masses, but you are correct about muting the existing object. I will fix that.

**Pull request merged**